microapps/ArWorldScaleApp/app/src/main/java/com/arcgismaps/toolkit/arworldscaleapp/screens/MainScreen.kt

```diff
@@ -299,6 +306,17 @@ fun MainScreen() {
 
                             else -> {}
                         }
+
+                        if (trackingError != null) {
```

Comment:
Could you add a comment here:
```suggestion
                        // If an error occurs during an initialized AR session, show a warning icon.
                        if (trackingError != null) {
```